.travis.yml

```diff
@@ -2,58 +2,14 @@ os: linux
 dist: focal
 language: minimal
 
-services:
-  - docker
-
 env:
-  # Testing QuickLogic toolchain on all OSes
```

Comment:
Why are these being dropped?

Reply:
I think this is because we were trying to jump over to GH actions which are not equivalent of Travis. This blew up anyhow because of GH's storage limit constraints so we're back to Travis - and so @fkokosinski let's get back to testing everything that was there originally, if it worked.